src/status_im/contexts/chat/messenger/composer/utils.cljs

```diff
@@ -86,12 +86,8 @@
        (calc-bottom-content-height images link-previews?))))
 
 (defn empty-input?
-  [text images link-previews? reply? audio?]
-  (and (empty? text)
-       (empty? images)
-       (not link-previews?)
-       (not reply?)
-       (not audio?)))
+  [{:keys [input-text images link-previews? reply audio edit]}]
+  (not (or (not-empty input-text) images link-previews? reply audio edit)))
```

Comment:
Should we check if images is `not-empty` too?

Reply:
I think it is being checked for as `images link-previews? reply audio edit` is wrapped in a `not`